tests/unit/services/intl.js

```diff
@@ -0,0 +1,75 @@
+import { moduleFor } from 'ember-qunit';
+import test from 'dummy/tests/ember-sinon-qunit/test';
+
+let service;
+
+moduleFor('service:intl', 'Unit | Service | intl', {
```

Comment:
I'd rather see more of an integration style test to get the whole flow.  I'm fine with keeping these unit tests though.